docker/Dockerfile

```diff
@@ -54,28 +61,6 @@ RUN dnf makecache && \
   dnf clean all && \
   rm -rf /var/cache/dnf/*
 
-RUN cd "/lib/systemd/system/sysinit.target.wants/"; \
-    for i in *; do [ $i = systemd-tmpfiles-setup.service ] || rm -f "$i"; done
-
-RUN rm -f /lib/systemd/system/multi-user.target.wants/* \
-    /etc/systemd/system/*.wants/* \
-    /lib/systemd/system/local-fs.target.wants/* \
-    /lib/systemd/system/sockets.target.wants/*udev* \
-    /lib/systemd/system/sockets.target.wants/*initctl* \
-    /lib/systemd/system/basic.target.wants/* \
-    /lib/systemd/system/anaconda.target.wants/*
-
-RUN systemctl set-default multi-user.target
-ENV init /lib/systemd/systemd
-
-# Build and install TPM 2.0 simulator
-WORKDIR ${TPM_HOME}
-RUN wget --content-disposition http://sourceforge.net/projects/ibmswtpm2/files/ibmtpm1119.tar.gz/download && \
-  tar -zxvf ibmtpm1119.tar.gz
-WORKDIR ${TPM_HOME}/src
-RUN make && \
-  install -c tpm_server /usr/local/bin/tpm_server
-
-VOLUME [ "/sys/fs/cgroup" ]
-
-ENTRYPOINT ["/lib/systemd/systemd"]
+# Install Rust
+RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
```

Comment:
If we're using Fedora, I'd strongly suggest to just add `rust` and `cargo` to the package list instead of installing it via Rustup.